src/xrCore/xrDebug_macros.h

```diff
@@ -68,7 +68,7 @@
     {\
         static bool ignoreAlways = false;\
         if (!ignoreAlways && !(expr))\
-            xrDebug::Fail(ignoreAlways, DEBUG_INFO, #expr);\
+            xrDebug::Fail(ignoreAlways, DEBUG_INFO, #expr, "Verifying failed");\
```

Comment:
Sorry for the delay in response.
According to "Assertion failed" message, it's better to say "Verification failed".
```suggestion
            xrDebug::Fail(ignoreAlways, DEBUG_INFO, #expr, "Verification failed");\
```

Reply:
"Assertion" is a well known thing whereas "verification" is something that makes no sense in this context. Like I said, VERIFY and R_ASSERT are basically the same and there is no need to introduce another message that just adds more confusion. It's not like you actually need to know whether the error was caused by VERIFY or R_ASSERT macro - you will have file name and line number anyway.

Reply:
They are almost the same, but VERIFY doesn't work for Release and it indicates non-critical errors. (in the most cases)

But... If we say `Even failed VERIFY should be threaten as an error` then it's a good point to not to change the message and leave it as `assertion failed`.